rasterio/_features.pyx

```diff
@@ -250,7 +250,7 @@ def _sieve(image, size, out, mask, connectivity):
         mask_mem_ds.close()
 
 
-def _rasterize(shapes, image, transform, all_touched):
+def _rasterize(shapes, image, transform, all_touched, merge_alg_add):
```

Comment:
@fredericbonifas so, up until now we've been using only the default `REPLACE` (see http://www.gdal.org/gdal__alg_8h.html). I'm 👍 on adding the `ADD` option. Will you please change `merge_alg_add` to `merge_alg`? It will take either `'REPLACE'` or `'ADD'`.